contracts/V1/StateChannelDiamondProxy/DisputeManagerFacet.sol

```diff
@@ -98,25 +98,29 @@ contract DisputeManagerFacet is StateChannelCommon {
         Dispute memory dispute,
         DisputeAuditingData memory disputeAuditingData
     ) public {
-       
-        // address challenger = msg.sender; // I dont think we should slash the auditor as they are like Polkadot fisherman
-
-        // (bool isAllAuditValid, address[] memory collectedSlashParticipants, bytes memory fraudProofErrorResult) = auditDispute(dispute, disputeAuditingData);
-
-        // if(isAllAuditValid) {
-        //     addOnChainSlashedParticipants(collectedSlashParticipants);
-        //     address[] memory returnedSlashParticipants = getOnChainSlashedParticipants();
-        //     emit DisputeChallengeResultWithError(dispute.channelId, isAllAuditValid, returnedSlashParticipants, fraudProofErrorResult);
-        // }
-        // else {
-        //     uint disputeLength = getDisputeLength(dispute.channelId);
-        //     DisputePair memory disputePair = DisputePair(dispute.disputeIndex, disputeLength-1);
-        //     onChainDisputePairs.push(disputePair);
-        //     addOnChainSlashedParticipants(collectedSlashParticipants);
-        //     address[] memory returnedSlashParticipants = getOnChainSlashedParticipants();
-        //     emit DisputeChallengeResultWithDisputePair(dispute.channelId, disputePair, isAllAuditValid, returnedSlashParticipants);
-        // }
-        
+        uint256 gasLimit = getGasLimit();
+        bytes memory data = abi.encodeCall(
+            DisputeManagerFacet.auditDispute,
+            (dispute, disputeAuditingData, block.timestamp)
```

Comment:
this is not the block.timestamp it's the block.timestamp when the dispute was created/committed. So you have to upload it so it will be checked against the commitment.